src/app/results/results.component.css

```diff
@@ -16,7 +16,6 @@
   text-overflow: ellipsis;
 }
 .infobox {
-  float:right!important;
 
 }
```

Comment:
@sudheesh001 I thought to keep `.infobox{}` so that if other properties can be added by other team members. I'll remove it.